joomla/joomla-cms
# Implement a check / notification for an invalid or missing access file

### Summary of Changes

This PR implements a check and notification for an invalid or missing access file that is accessed in both core and 3rd party extensions via the JHelperContent::getActions method.

In the current implementation, depending on the error reporting level, an invalid argument warning would be either shown to the user or written to the php error log file.
### Testing Instructions
- Set the error reporting level of your (test) installation to maximum (Global Configuration - Server Settings - Error Reporting)
- Head over to the file system and rename an access.xml for e.g. administrator/components/com_banners to access.xml_
- Open the banners component in the backend. You should see something similar to this:
  ![pr_banner_before_patch](https://cloud.githubusercontent.com/assets/1304616/19808989/82f79e58-9d27-11e6-9dd3-10f5c1c45111.jpg)

In a normal running website, the error reporting level would be set to "System Default" (reporting). The only indication for an invalid or missing access file, is just a few missing toolbar buttons.
- Apply the patch. You should see an error message like the one in the screenshot below.
  ![pr_banner_after_patch](https://cloud.githubusercontent.com/assets/1304616/19809005/90e82e42-9d27-11e6-94a8-94239ce27d9f.jpg)
